orchestra/static/orchestra/project/js/controllers.js

```diff
@@ -0,0 +1,870 @@
+(function () {
+  // 'use strict';
+
+  angular
+    .module('orchestra.project.controllers')
+    .controller('ProjectController', ProjectController);
+
+  ProjectController.$inject = ['$location', '$scope', '$routeParams', '$http', '$sce',
+                            '$compile', '$modal', '$timeout', 'dataService', 'orchestraApi'];
+
+  function ProjectController($location, $scope, $routeParams, $http, $sce,
+                          $compile, $modal, $timeout, dataService, orchestraApi) {
+    // var vm = this;
+    vm = $scope;
+    $scope.activate = function() {
+      // Prevent initial animation of vis load
+      d3.select('.project-management').style('opacity', 0);
+      $scope.vis = new projectVis($routeParams.projectId, '.project-management .svg-wrapper');
+      $scope.vis.update();
+      d3.select('.project-management').transition()
+        .delay(500)
+        .style('opacity', 1);
+    };
+    $scope.activate();
+
+    function projectVis(projectId, parentSelector) {
+      var vis = this;
+
+      vis.projectId = projectId;
+      vis.parentContainer = d3.select(parentSelector);
+
+      vis.options = {
+        'scaleHeight': 40,
+        'barHeight': 30,
+        'lanePadding': {'top': 30, 'bottom': 25},
+        'marginLeft': 100,
+        'marginRight': 10,
+      }
+
+      vis.relativeTime = true;
+      vis.scaleWidth = 1350;
+
+      vis.crosshair = new crosshair();
+      vis.actions = new actions();
+      vis.utils = new utils();
+      vis.tasks = new tasksVis();
+      vis.assignments = new assignmentsVis();
```

Comment:
style: you're using `new` and `class`-like patterns for creating each of these objects.  But classes are typically used for encapsulating state.  A lot of the code for things like `assignments` and other variables here is just a nice modularization of your logic.  Since the modules don't maintain/encapsulate internal state, it would be better (stylistically) to avoid using `new`, and instead follow the module/singleton pattern: http://stackoverflow.com/a/1479341

it won't involve crazy changes---basically, rather than assigning functions on `this` in each module, you return an object/dict with all of the public methods you want to export.